docs/ides.md

```diff
@@ -88,11 +88,11 @@ After installing Gateway on your local system, you may connect to a Coder worksp
       already.
 8. Click "Download and Start IDE" to connect.
 
-| Version   | Status      | Notes                                                      |
-|-----------|-------------|------------------------------------------------------------|
-| 2021.3.2  | Working     |                                                            |
-| 2022.1.1  | Working     | Windows clients are unable to connect to Linux workspace   |
-| 2022.2 RC | Not working | [GitHub Issue](https://github.com/coder/coder/issues/3125) |
+| Version   | Status  | Notes                                                    |
+|-----------|---------|----------------------------------------------------------|
+| 2021.3.2  | Working |                                                          |
+| 2022.1.1  | Working | Windows clients are unable to connect to Linux workspace |
+| 2022.2 RC | Working | Version >= 222.3345.108                                  |
```

Comment:
Not sure if we should leave a note here since our docs are version-controlled, but only worked for me in main (Coder >=v0.8.2)

Reply:
We should be deploying docs corresponding to releases.